cmd/config.go

```diff
@@ -234,6 +238,11 @@ func config(ctx *cli.Context) error {
 				format.TrashDays = new
 				trash = true
 			}
+		case "dir-stats":
+			if new := ctx.Bool(flag); new != format.DirStats {
+				msg.WriteString(fmt.Sprintf("%10s: %t -> %t\n", flag, format.DirStats, new))
+				format.DirStats = new
```

Comment:
tell user what to do (calculate the stats for existing dirs) after dirStat is enabled

Reply:
achieve consensus, no need